src/EPR.Calculator.API.Data/obj/Debug/net8.0/EPR.Calculator.API.Data.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -0,0 +1,13 @@
+is_global = true
```

Comment:
The obj directory can be added to .gitignore.

Reply:
Done